tests/security/cc/ip_eb_tables.pm

```diff
@@ -24,27 +25,33 @@ sub run {
 
     select_console 'root-console';
 
-    # Configure bridge for ip_eb_tables workaround
-    assert_script_run("cat > $f_ifcfg_br0 <<'END'\n$br0_config\nEND\n( exit \$?)");
+    # In system role is CC, the bridge is set up automatically in x86 and arm,
+    # so we only need to configure the bridge network for s390
```

Comment:
it should be **s390x** here

Reply:
In commit message should also be changed to **s390x** too.

Reply:
fixed